### PR TITLE
Prevent app follows SAML redirections

### LINKED ISSUE #758: Alert Error server connection after login in SAML servers
### Steps to reproduce
1. In a SAML server, wait until session expires
2. Input credentials
### Expected behaviour

The session starts with no error messages 
### Actual behaviour

After input credentials, a message `It is not possible to connect to the server at this time` is always displayed
### Server configuration

{"installed":true,"maintenance":false,"version":"9.1.0.16","versionstring":"9.1.0","edition":"Enterprise"}
### Client

iPad v8.4 & v9
iPhone v9

### PR DESCRIPTION
**Pull request by davivel**
Requires https://github.com/owncloud/ios-library/pull/146

Fixes https://github.com/owncloud/enterprise/issues/1775

Fixes #758

### CODE REVIEW

File: Owncloud iOs Client/Branding/Customization.h [hunk 1]
```diff
@@ -41,7 +41,7 @@
 #define k_is_oauth_active NO
 
 //Have SAML active
-#define k_is_sso_active NO
+#define k_is_sso_active YES
```
Comment (jesmrec) [inline Owncloud iOs Client/Branding/Customization.h]: it is not recomendable to hardcode this @davivel

**Comment by jesmrec** [comment]
The problem in #758 keeps on happening, unfortunately

I could reproduce it easily using iPad and servers 9.1.3 and 10.0.

CC @nasli @davivel

File: Owncloud iOs Client/Branding/Customization.h (hunk 1 above)
Comment (davivel) [inline Owncloud iOs Client/Branding/Customization.h]: Sure! :S

**Comment by davivel** [comment]
OK, @ggdiez found something else that could be needed, both for https://github.com/owncloud/enterprise/issues/1775 and https://github.com/owncloud/enterprise/issues/1768

Let's close this one for the moment.

**Pull request closed without merging**